### PR TITLE
Fix controls for Plone 5

### LINKED ISSUE #22: Setting show controls breaks the ticker in Plone 5
do not show controls:

selection_018

show controls:

selection_017

### PR DESCRIPTION
**Pull request by rodfersou**
closes #22

### CODE REVIEW

File: webpack/webpack.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
     filename: 'newsticker.js',
     path: '../src/collective/newsticker/browser/static',
     libraryTarget: 'umd',
-    publicPath: '++resource++collective.newsticker/',
+    publicPath: '/++resource++collective.newsticker/',
```
Comment (rodfersou) [inline webpack/webpack.config.js]: needed by Plone 5; already tested with Plone 4 and work too.

File: webpack/app/newsticker.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ $.fn.ticker = function(options) {
       // add the controls to the DOM if required
       if (opts.controls) {
         // add related events - set functions to run on given event
-        $(settings.dom.controlsID).live('click mouseover mousedown mouseout mouseup', e => {
+        $('#ticker-wrapper').on('click mouseover mousedown mouseout mouseup', settings.dom.controlsID, e => {
```
Comment (rodfersou) [inline webpack/app/newsticker.js]: This is the new way to create live events; attach the event on one tag above that will not be modified by javascript (and then lose the event registration).

**Comment by rodfersou** [comment]
@hvelarde green

**Comment by hvelarde** [comment]
@rodfersou is not working; as you can see in the image.

selection_017

BTW, @agnogueira there are two tones of gray in the ticker.

**Comment by rodfersou** [comment]
@hvelarde here it works
image

**Pull request merged**